Makefile

```diff
@@ -290,7 +291,8 @@ test_issues: $(SASSC_BIN)
 	$(RUBY_BIN) $(SASS_SPEC_PATH)/sass-spec.rb -c $(SASSC_BIN) $(LOG_FLAGS) $(SASS_SPEC_PATH)/spec/issues
 
 clean-objects:
-	$(RM) lib/*.a lib/*.so lib/*.dll lib/*.la
+	-$(RM) lib/*.a lib/*.so lib/*.dll lib/*.la
+	-$(RMDIR) lib
 clean: clean-objects
```

Comment:
Why is there a dash before the variables/command? And I somehow would rather put the rmdir into the clean target itself, but I guess that's just a personal preference, but maybe you can see where I'm going ...